src/app/calendar/page.tsx

```diff
@@ -1,16 +1,18 @@
 import { Metadata } from "next";
+import SampleCalendar from "./samplePage";
 import React from "react";
 
-const Calendar: React.FC<{}> = () => {
+export const metadata: Metadata = {
+    title: "Calendar",
+};
+
+const CalendarPage: React.FC<{}> = () => {
     return (
         <main>
-            <h1> Calendar Page </h1>
+            <h1>Calendar Page</h1>
+            <SampleCalendar />
```

Comment:
Can you not change the Calendar page? if it's just a sample then when we make the calendar page then we will make it 'correct' - I'm happy with a sample page as a 'reference' for now but would prefer it to just be available as a component, not actually in use on the live (test) website!